wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -2044,3 +2044,28 @@ to reset this device and log in again";
 // MARK: - Time
 
 "time.just_now" = "Just now";
+
+//MARK: - Device
+"device.mls.thumbprint" = "MLS Thumbprint";
+"device.proteus.id" = "Proteus ID";
+"device.current" = "CURRENT DEVICE";
+"device.active" = "ACTIVE";
```

Comment:
We have strings `Active` and `CURRENT DEVICE`, you don't need a new one. This part of the UI already exists

Reply:
We have `Active` but not `Current Device`.